src/SFA.DAS.ApplyService.Web/Views/Application/PagesWithSections.cshtml

```diff
@@ -25,25 +25,23 @@
             else
             {
                 <h1 class="govuk-heading-xl">@Model.Title</h1>
-                <ul class="govuk-list">
+
+                <ol class="app-task-list">
                     <li>
-                        <ul class="app-task-list govuk-!-margin-bottom-7">
+                        <h2 class="app-task-list__section">@Html.Raw(pagesToCheck[0].Title)</h2>
+                        <ul class="app-task-list__items app-task-list__items--no-number">
```

Comment:
Have we checked all of this displays correctly as it's feels like very complex logic (which could probably benefit from being re-worked). From what I understand it's looking through all pages to check, and when it is a 'new page (i.e. a new title)' it dumps out the new title and puts stuff into a new unordered list 🤷‍♂️

Reply:
Seems to display correctly but I'm happy for the logic to be re-worked at a later date.